ci_env.yml

```diff
@@ -0,0 +1,16 @@
+name: biomechanical-analysis-framework
+channels:
+  - conda-forge
+  - robotology
```

Comment:
It seems all the packages you use are from the conda-forge channel, we can probably remove this.

Reply:
Removed!